test/models/extract_text_from_paper_job_test.rb

```diff
@@ -0,0 +1,19 @@
+require 'test_helper'
+
+class ExtractTextFromPaperJobTest < ActiveSupport::TestCase
+  test 'keep hyphen at wordwrap if it is a name' do
+    text = "Beate Walter-\n" +
+           "Rosenheimer"
+    # result = "Beate Walter-Rosenheimer\n"
+    result_text = ExtractTextFromPaperJob.clean_text(text)
+    assert_equal text, result_text
+  end
+
+  test 'remove hyphen at wordwrap' do
+    text = "be-\n" +
+           "pflanzt"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.